share/functions/__fish_systemctl_services.fish

```diff
@@ -3,15 +3,34 @@ function __fish_systemctl_services
     # We don't want to complete with ANSI color codes
     set -lu SYSTEMD_COLORS
 
-    if type -q systemctl
-        if __fish_contains_opt user
```

Comment:
thanks for the fixes. I'll push a version with a cleaned-up Git history,
and I'll remove the $argv since I don't think we can explain why it's necessary (or can we?)